components/workspaces/staging/base/server/config/server/kustomization.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- deployment.yaml
+- service.yaml
```

Comment:
did `service.yaml` get deleted?

Reply:
no, it's there. It's easier to find it if you collapse the `proxy-config` folder.